free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Mudders get a lot of email. 
+Many people subscribe to dorm chats and occasionally receive
+email with interesting content but a lot of times they get useless
+emails (these range from counting threads, to emails asking for rides from
+airports, and to ASHMC senate summary emails).
+
+As an example, these are a series of emails sent to a certain dorm's chat
+list
```

Comment:
While I don't know what a counting thread is (is that a Mudd thing?) I can definitely attest to the pain of unnecessary emails.